src/index.html

```diff
@@ -5,8 +5,388 @@
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog">
+    <header class="catalog-header">
+      <a href="index.html" class="catalog-header__logo">
+        <img
+          class="catalog-header__logo-img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__links is-active" href="index.html">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__links"
+            href="index.html"
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__links" href="index.html">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog__main">
+      <div class="catalog__wrapper">
+        <div class="card" data-qa="card">
+          <img
+          class="card__img"
+          src="/src/images/imac.jpeg"
+          alt="imac"></img>
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__code">
+            Product code: 195434
+          </p>
+          <div class="card__wrapper card__wrapper--reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__review">
+              Reviews: 5
+            </p>
+          </div>
+          <div class="card__wrapper card__wrapper--price">
+            <p class="card__price-key">
+              Price:
+            </p>
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+          <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+          >
+          buy
+          </a>
+        </div>
+        <div class="card" data-qa="card">
```

Comment:
Add the blank line between the cards
```suggestion
        </div>
        
        <div class="card" data-qa="card">
```